frontend/src/components/Collapsible/Collapsible.jsx

```diff
@@ -5,29 +5,34 @@ import "./index.less";
 
 const { Title } = Typography;
 
-const Collapsible = ({ initiallyCollapsed, title, children }) => {
+const Collapsible = ({ initiallyCollapsed, title, children, headerStyle }) => { // eslint-disable-line
   const [isCollapsed, setIsCollapsed] = useState(initiallyCollapsed);
 
   const toggleCollapse = () => {
     setIsCollapsed(!isCollapsed);
   };
 
   return (
-    <>
-      <div className="collapsible-header" onClick={toggleCollapse} role="none">
+    <div className="collapsible">
+      <div className="collapsible-header" onClick={toggleCollapse} role="none" style={headerStyle}>
         <RightOutlined
           className={(isCollapsed)
             ? "collapsible-button collapsible-button-collapsed"
             : "collapsible-button"}
         />
-        <Title level={3} className="text">
-          {title}
-        </Title>
+        {(typeof title === "string" || title instanceof String)
```

Comment:
Why are we doing two checks here? Could we just do either 'typeof' or 'instanceof' rather than both?

Reply:
i guess